lab_1_classify_by_unigrams/main.py

```diff
@@ -21,8 +22,17 @@ def tokenize(text: str) -> list[str] | None:
     In case of corrupt input arguments, None is returned
     """
 
+    tokens = []
+    if isinstance(text, str):
```

Comment:
Swap the condition. Make function return None instantly if type is wrong. Dont place all code in if statement.